using-mobile-extensions/mobile-core/mobile-core-release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release Notes
 
+### September 10, 2020
+
+#### iOS Core 2.7.4
+
+* Fixed an intermitted issue where third party extensions may experience crashes after unregistration when using the `ACPExtensionAPI`.
+
+#### iOS Lifecycle 2.1.2
+
+* Added previous application Id and OS version info to lifecycle event data.
```

Comment:
We should also update the Lifecycle event page for this change: https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle/lifecycle-event-reference#lifecycle-data-content-response

Reply:
it is inside the `lifecyclecontextdata` map, so unless we want to show all the possible key/value pairs, then we are good for now.